attributes/default.rb

```diff
@@ -569,3 +569,6 @@
 default['hops']['docker']['testconnector']['image']['name']    = "testconnector"
 default['hops']['docker']['testconnector']['image']['version'] = "0.1"
 default['hops']['docker']['testconnector']['download_url'] = "#{node['download_url']}/testconnector/#{node['hops']['docker']['testconnector']['image']['version']}/testconnector.tgz"
+
+default['hops']['docker']['hopsfsmount-seccomp-profile'] = "/etc/docker/hopsfsmount_seccomp_profile.json"
+default['hops']['docker']['hopsfsmount-apparmor-profile'] = "hopsworks-hopsfsmount-docker"
```

Comment:
Sorry something I forgot, this has be exported in `metadata.rb` in order to change it from the cluster definition